.travis.yml

```diff
@@ -5,47 +5,33 @@ language: bash
 services:
   - docker
 
-env:
-  - HIVE_VERSION=2.3.6 RUN_PUSH=true RUN_DEPLOY=true # actual stable version
```

Comment:
Skusali sme to oproti viacerym hive verziam, nevedeli sme aku verziu ma CSAS.
Teraz je dolezitejsie aby sme to vedeli ostestovat oproti Kerberos, ... ak by som chcel podporovat aj Kerberos a aj viacero verzii, tak je to zbytocna robota na dalsich niekolko dni.